instance/models/instance.py

```diff
@@ -210,6 +224,11 @@ def modified(self):
         """ Get this instance's modified date, which is stored in the InstanceReference """
         return self.ref.modified
 
+    @property
+    def instance_purpose(self):
+        """ Get this instance's purpose, which is stored in the InstanceReference"""
+        return self.ref.get_instance_purpose_display()
```

Comment:
The get_FOO_display method returns the human_readable value of the provided choice. [Docs](https://docs.djangoproject.com/en/1.11/ref/models/instances/#django.db.models.Model.get_FOO_display).